lib/actions/local/configure.coffee

```diff
@@ -95,8 +66,48 @@ inquirerOptions = (data) -> [
 	}
 ]
 
+# Since these paths are used by `resin-image-fs`, we can safely
+# assume they are always UNIX style.
+joinPath = (path) -> '/' + path.join('/')
+
+loadReconfix = (target) ->
+	{ Reconfix } = require('reconfix-preview')
```

Comment:
Can we export `Reconfix` directly, rather than exporting an object containing a `Reconfix` property? I should ideally be able to do `Reconfix = require('reconfix')`